README.md

```diff
@@ -10,12 +10,13 @@ Documentation: https://reddit-experiments.readthedocs.io/
 Install the library:
 
 ```console
-$ pip install reddit-experiments
+$ pip install reddit-experiments reddit-v2-events>=2.8.2
```

Comment:
Users can't install this—we only have a 2-year-old package in [pypi](https://pypi.org/project/reddit-v2-events/#files) that doesn't have any code in it.

It's not a great idea to put internal reddit instructions on public github. I'm not sure if an event logger exists in any of the public reddit repos, but if not we should probably just omit this.

Reply:
given that there's only [1 fork](https://github.com/reddit/experiments.py/forks) (from a reddit employee), should we be concerned?

I don't think we're getting much open-source usage of the repo and this way employees will know where to get the event logger during the migration--otherwise we'll just field more questions, or worse--people will continue copy/pasting the code.

Reply:
I added a comment clarifying that the package is reddit internal.